torchvision/models/detection/faster_rcnn.py

```diff
@@ -300,6 +300,8 @@ def fasterrcnn_resnet50_fpn(pretrained=False, progress=True,
     """
     Constructs a Faster R-CNN model with a ResNet-50-FPN backbone.
 
+    Reference: `"Faster R-CNN: Towards Real-Time Object Detection with Region Proposal Networks" <https://arxiv.org/abs/1506.01497>`_. # noqa
```

Comment:
instead of adding `# noqa` we should just break this line to be shorter, it won't affect the html rendering

Reply:
Oh, will do that then. Thanks!